### ISSUE TITLE
How do I correctly set up basil.js for development?

### ISSUE THREAD

**Issue by trych** [basiljs/basil.js#52]
Hi there,
so I would like to start contributing now, however, as I am new to Github, I have issues, at wrapping my head around how to set it up correctly. I read the Wiki, I forked and then cloned the project to my local drive, and set it up in Sublime, that all works fine.

However, what I do not understand: Do I have to install my "development version" of basil.js _alongside_ the regularly installed version or do I have to install it _instead_ of the regularly installed version? Specifically, I do not understand how to hook up my cloned version into InDesign in a way that I can run any basil scripts. As they all require the basil.js file to sit exactly at `USER_FOLDER/Documents/basiljs/bundle/basil.js`, how does this work with my cloned repository (which is by default a folder named `basil.js` that does not contain a bundle folder, i.e. the result is that the basil.js file is not at `USER_FOLDER/Documents/basiljs/bundle/basil.js`)?

**tldr**: Since my "development basil.js" file is not at `USER_FOLDER/Documents/basiljs/bundle/basil.js`, I cannot run any of my "development basil.js" scripts, so how do I work around that?

Thanks!

Edit: This all refers to OS X, in case that matters.

**Comment by trych** [comment]
Alright, figured it out myself, you have to navigate to the `USER_FOLDER/Documents/basiljs/` location and then do `git clone https://github.com/basiljs/basil.js.git bundle`. Did not know that adding a name, will prevent the local repository folder to be named after the repository name. So, now it sits at the correct place and works. Issue can be closed.

Moving on.
